WordPress/Classes/ViewRelated/Views/List/ListTableViewCell.swift

```diff
@@ -135,13 +138,13 @@ private extension ListTableViewCell {
         indicatorView.layer.cornerRadius = indicatorWidthConstraint.constant / 2
 
         // title label
-        titleLabel.font = Style.plainTitleFont
-        titleLabel.textColor = Style.titleTextColor
+        titleLabel.setStyle(.bodySmall(.regular))
+        titleLabel.textColor = .DS.Foreground.primary
         titleLabel.numberOfLines = Constants.titleNumberOfLinesWithSnippet
 
         // snippet label
-        snippetLabel.font = Style.snippetFont
-        snippetLabel.textColor = Style.snippetTextColor
+        snippetLabel.setStyle(.bodySmall(.regular))
+        snippetLabel.textColor = .DS.Foreground.secondary
         snippetLabel.numberOfLines = Constants.snippetNumberOfLines
     }
```

Comment:
⚠️ Same comment as this [one](https://github.com/wordpress-mobile/WordPress-iOS/pull/22545/files#r1480051482). The `ListTableViewCell` is used in `CommentsViewController`

Reply:
These don't really change the existing values but use DesignSystem values instead. I think we can keep it like this?

Reply:
Ah okey, yeah we can keep these changes 👍